TypeCobol/packages.config

```diff
@@ -5,6 +5,7 @@
   <package id="Antlr4.Runtime" version="4.6.6-rc001" targetFramework="net45" />
   <package id="Castle.Core" version="4.1.1" targetFramework="net45" />
   <package id="JetBrains.Annotations" version="11.0.0" targetFramework="net45" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net45" />
```

Comment:
Razor is not used in the TypeCobol project, why is the package reference added ?

Reply:
Ok This reference was added before the complete removal of Razor which has been done in v1.3.